modules/schemas/src/main/resources/contexts/nexus/core/error/v0.1.0.json

```diff
@@ -3,7 +3,7 @@
     "{{base}}/contexts/nexus/core/standards/v0.1.0",
     {
       "code": {
-        "@code": "nxv:code"
+        "@id": "nxv:code"
```

Comment:
we can even have it like `"code": "nxv:code"`, right?

Reply:
Yes. I wanted to be consistent with the other prefix mappings.